src/patternfly/_variables.scss

```diff
@@ -1,5 +1,82 @@
 :root {
   // Colors
+  // Global palette
+  --pf-global--Color--black-100: #{$pf-color-black-100};
```

Comment:
Should we be using these variables like `--pf-global--Color--black-100` now in places where we used values like `#{$pf-color-black-100}` ? i.e. charts?

Reply:
Yeah, great idea! I'll update them in this PR.

Reply:
Only one I didn't update is `--pf-c-about-modal-box__close--c-button--hover--BackgroundColor: #{rgba($pf-color-black-1000, .4)};` since passing the css var to sass's `rgba()` function doesn't work.